server/base/src/main/java/org/apache/accumulo/server/util/ChangeSecret.java

```diff
@@ -69,19 +70,21 @@ public static void main(String[] args) throws Exception {
     argsList.add("--old");
     argsList.add("--new");
     argsList.addAll(Arrays.asList(args));
-    opts.parseArgs(ChangeSecret.class.getName(), argsList.toArray(new String[0]));
+    try (TraceScope clientSpan = opts.parseArgsAndTrace(ChangeSecret.class.getName(),
```

Comment:
The clientSpan variable is never used. You could take out the try with resources and call `clientSpan.close();` at the end.  Which is almost the same thing but IDEs and code analysis tools will at least recognize the use of clientSpan.  IntelliJ gives me this warning FYI.  I think the only difference is if and error is thrown the span wouldn't get closed though.

Reply:
The design of the API is to use this with try-with-resources. I had to suppress the warning from javac. We could avoid warnings by referencing it by annotating the spans better. But, a lot of our problems throughout our code involved manual closing of these spans.... and trace bugs due to failure to do so. The try-with-resources construct, as the trace API intends, protects against those kinds of errors. Can you suppress this warning type in IntelliJ?